app/Libraries/Storage/CoreUploadedFile.php

```diff
@@ -15,4 +15,9 @@ public function getFullFileName()
     {
         return sha1("{$this->getClientOriginalName()}.{$this->getClientOriginalExtension()}");
     }
+
+    public function getFilePathName()
+    {
+        return $this->getFullFileName() . '.' . $this->extension();
```

Comment:
Consistency with the method above with respect to string construction please me duck